src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ApprenticeController.cs

```diff
@@ -1,9 +1,9 @@
 ﻿using MediatR;
 using Microsoft.AspNetCore.Mvc;
 using SFA.DAS.ApprenticeCommitments.Application.Commands.CreateRegistrationCommand;
-using SFA.DAS.ApprenticeCommitments.Application.Queries.ApprenticeQuery;
 using System;
 using System.Threading.Tasks;
+using SFA.DAS.ApprenticeCommitments.Application.Queries.ApprenticeshipsQuery;
```

Comment:
Obsessive-compulsive shiver from the out-of-order change ;)  Doesn't resharper order these automatically?